specification.markdown

```diff
@@ -386,14 +386,13 @@ The path defines the allowed storage locations of the EBML Element within an EBM
 The path attribute is REQUIRED.
 
 ```
-EBMLFullPath          = [EBMLParentPath] EBMLElementPath
-EBMLParentPath        = EBMLFixedParent EBMLLastParent
-EBMLFixedParent       = *(EBMLPathAtom)
+EBMLFullPath          = [EBMLParentPath] [EBMLGlobalParent] 
+                        EBMLElementPath
+EBMLParentPath        = 1*(EBMLElementPath)
 EBMLElementPath       = EBMLPathAtom / EBMLPathAtomRecursive
+EBMLPathAtomRecursive = "(" EBMLPathAtom ")"
 EBMLPathAtom          = PathDelimiter EBMLAtomName
-EBMLPathAtomRecursive = "(1*(" EBMLPathAtom "))"
-EBMLLastParent        = EBMLPathAtom / EBMLGlobalParent
-EBMLGlobalParent      = "(" GlobalParentOccurence "\)"
+EBMLGlobalParent      = "(" GlobalParentOccurence ")"
```

Comment:
I would prefer not to. Because changing this sentence would be tricky:

> The GlobalParentOccurence part is interpreted as an ABNF Variable Repetition

While `x*y` is a valid repetition in ABNF, `"("x*y")"` or `(x*y)` are not.
Many of the items are split so we can define them in the text.